AzureActiveDirectory/manifest.json

```
@@ -17,6 +17,16 @@
                 "title": "Client Secret",
                 "description": "Client Secret associated with the registered application. Admin Consent has to be granted to the application for it to work.",
                 "type": "string"
+            },
+            "username": {
+                "title": "Client UserName",
+                "description": "Client username account to use reset password action. the account should have admin permissions to reset password",
+                "type": "string"
+            },
+            "password": {
+                "title": "New Password",
+                "description": "New password for the target user. the account should have admin permissions to reset password",
```

I prefer to talk about delegated account.
```suggestion
                "title": "Delegated account username",
                "description": "The username of the delegated account used for some administrative tasks (eg: reset password)",
                "type": "string"
            },
            "password": {
                "title": "Delegated account password",
                "description": "The password of the delegated account used for some administrative tasks (eg: reset password)",
```

---

Yes you right, thank you